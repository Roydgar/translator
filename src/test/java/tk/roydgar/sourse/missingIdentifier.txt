program myprog;
const  = 10;

begin
end.