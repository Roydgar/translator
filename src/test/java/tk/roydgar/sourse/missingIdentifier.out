   1    1        301    program
   1    9        401    myprog
   1   15          3    ;
   2    1        304    const
   2    8          0    =
   2   10        501    10
   2   12          3    ;
   4    1        302    begin
   5    1        303    end
   5    4          5    .

Tree:
<signal-program>
    <program>
        PROGRAM
        <procedure-identifier>
            <identifier>
                myprog
        ;
        <block>
            <declarations>
                <constant-declaration>
                    <empty>
            BEGIN
            <statement-list>
                <empty>
            END
        .
